### PR TITLE
New terms for annotating scRNAseq datasets

### PR DESCRIPTION
**Pull request by gouttegd**
This PR adds a set of new terms that will be needed to annotate scRNAseq datasets for their inclusion into FlyBase.

It also assigns an ID range to me, since it's my first contribution to FBcv.

related to #138

### CODE REVIEW

File: src/ontology/fbcv-edit.obo [hunk 1]
```diff
@@ -8908,6 +8908,88 @@ is_a: FBcv:0000014 ! structural qualifier
...
+is_a: FBcv:0003050 ! assay type
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-07-26T12:19:45Z
+
+[Term]
+id: FBcv:0009002
+name: cell clustering analysis
+namespace: result_type
+def: "A result that groups individual cells into populations sharing a similar transcriptional profile." [PMID:28821273]
+is_a: FBcv:0003082 ! result type
```
Comment (Clare72) [inline src/ontology/fbcv-edit.obo:8935]: I think this is an 'analysis'.
some other stuff under 'result type', e.g. 'differential gene expression analysis' and 'expression clustering' should probably also be under 'analysis' too.

**Review by Clare72 — CHANGES_REQUESTED** [review]

**Comment by Clare72** [comment]
Don't worry about failing checks, looks there was a problem downloading CHEBI (happens sometimes) - it will probably succeed next time (or you can just restart the travis build if it happens again).

File: src/ontology/fbcv-edit.obo [hunk 2]
```diff
@@ -8908,6 +8908,88 @@ is_a: FBcv:0000014 ! structural qualifier
 created_by: http://orcid.org/0000-0002-1373-1705
 creation_date: 2020-12-14T14:39:35Z
 
+[Term]
+id: FBcv:0009000
+name: single-cell RNA-Seq
+namespace: assay_type
+def: "A method for quantifying the transcriptome of individual cells, in which transcripts isolated from single cells are subjected to high-throughput sequencing and mapping to a reference genome." [PMID:28821273]
+xref: OBI:0002631
+is_a: FBcv:0003050 ! assay type
```
[the remark this answers is not in the document]
Comment (gouttegd) [reply src/ontology/fbcv-edit.obo:8917]: If we go that route, shouldn't `short RNA-Seq` also be a child a `RNA-Seq`?

[the remark this answers is not in the document]
Comment (Clare72) [reply src/ontology/fbcv-edit.obo]: yes, it probably should

[the remark this answers is not in the document]
Comment (gouttegd) [reply src/ontology/fbcv-edit.obo]: This is done in the last commits I pushed. In the same vein I also moved all the `result type` terms that can reasonably be considered to refer to an "analysis" under the `analysis` term, as suggested in your comment below.

**Review by Clare72 — APPROVED** [review]

**Pull request merged**